# DP: recurse into xorgxrdp/ as well (make into proper configure option?)
# DP: also, actually install keymaps shipped (should be upstreamed)
# DP: TODO also sync xrdp/xrdp_keyboard.ini

--- a/Makefile.am
+++ b/Makefile.am
@@ -27,4 +27,5 @@ SUBDIRS = \
   instfiles \
   genkeymap \
   xrdpapi \
+  xorgxrdp \
   $(XRDPVRDIR)
--- a/instfiles/Makefile.am
+++ b/instfiles/Makefile.am
@@ -14,10 +14,14 @@ startscript_DATA = \
   km-040c.ini \
   km-0410.ini \
   km-0411.ini \
+  km-0414.ini \
   km-0415.ini \
+  km-0416.ini \
   km-0419.ini \
   km-041d.ini \
   km-0807.ini \
+  km-080c.ini \
+  km-0813.ini \
   km-0816.ini \
   km-e0010411.ini \
   km-e0200411.ini \
--- a/xrdpapi/Makefile.am
+++ b/xrdpapi/Makefile.am
@@ -9,6 +9,7 @@ AM_CFLAGS = \
   $(EXTRA_DEFINES)
 
 INCLUDES = \
+  -I$(top_srcdir)/common \
   $(EXTRA_INCLUDES)
 
 lib_LTLIBRARIES = \
